src/components/app/content/cards/card/index.js

```diff
@@ -1,16 +1,21 @@
-import React, {Fragment, useState} from 'react';
+import React, {Fragment, useEffect, useState} from 'react';
 import './index.css'
 
 const classNames = require('classnames');
 
-function Card() {
+function Card({readonly}) {
```

Comment:
Ideally your app should be compiled with no warnings:

![image](https://user-images.githubusercontent.com/52723192/105766569-fbc38900-5f6a-11eb-80ec-71a896c09ff5.png)